### PR TITLE
fix noop overwrite image service regression

### PR DESCRIPTION
**Pull request by alexanderniebuhr**
## Changes

- for context see messages after: https://discord.com/channels/830184174198718474/845430950191038464/1165031078722011238
- I've added PRs to the two known edge runtimes:

- https://github.com/denoland/deno-astro-adapter/pull/7
- https://github.com/withastro/adapters/pull/33

### CODE REVIEW

**Comment by alexanderniebuhr** [comment]
Current dependencies on/for this PR:
* main
  * **PR #8880** <a href="https://app.graphite.dev/github/pr/withastro/astro/8880" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/withastro/astro/8880?utm_source=stack-comment).

**Comment by alexanderniebuhr** [comment]
!preview noop-imageservice

File: .changeset/witty-waves-rhyme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+Removes image service override, in favor to moving that logic into adapters and fix the regression it introduced
```
Comment (sarah11918) [inline .changeset/witty-waves-rhyme.md:5]:
```suggestion
Moves the logic for overriding the image service out of core and into adapters.  Also fixes a regression where  a valid `astro:assets` image service configuration could be overridden.
```
Maybe something like this? (I know the other adapter 33 PR has not finalized its changeset wording yet. So, whatever you decide there might affect what you put here.)

File: .changeset/witty-waves-rhyme.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+Moves the logic for overriding the image service out of core and into adapters.  Also fixes a regression where  a valid `astro:assets` image service configuration could be overridden.
```
Comment (sarah11918) [inline .changeset/witty-waves-rhyme.md:5]:
```suggestion
Moves the logic for overriding the image service out of core and into adapters. Also fixes a regression where a valid `astro:assets` image service configuration could be overridden.
```

Comment (sarah11918) [reply .changeset/witty-waves-rhyme.md:5]: Just a couple of double spaces here!

**Review by sarah11918 — APPROVED** [review]
Just fixed some spaces in the changeset. Otherwise, nice job @alexanderniebuhr .... Docs out!

**Pull request merged**